### PR TITLE
chore: Change default config value

### PR DESCRIPTION
**Pull request by gitferry**
This PR changed the default config values, ensuring consistent with [devnet](https://github.com/babylonchain/devnet-k8s/blob/main/flux/finality-provider/helmrelease.yaml).

This PR removed `UnbondingSigSubmissionInterval` as it is not applicable. cc @filippos47

### CODE REVIEW

File: finality-provider/config/config.go [hunk 1]
```diff
@@ -24,21 +24,21 @@ const (
 	defaultFinalityProviderKeyName        = "finality-provider"
 	DefaultRPCPort                        = 12581
 	defaultConfigFileName                 = "fpd.conf"
-	defaultNumPubRand                     = 100
-	defaultNumPubRandMax                  = 100
-	defaultMinRandHeightGap               = 10
-	defaultStatusUpdateInterval           = 5 * time.Second
-	defaultRandomInterval                 = 5 * time.Second
-	defautlUnbondingSigSubmissionInterval = 20 * time.Second
+	defaultNumPubRand                     = 150
```
Comment (vitsalis) [inline finality-provider/config/config.go:27]: Let's have this to be 100?

File: finality-provider/config/config.go [hunk 2]
```diff
@@ -24,21 +24,21 @@ const (
...
+	defaultNumPubRandMax                  = 200
+	defaultMinRandHeightGap               = 50
```
Comment (vitsalis) [inline finality-provider/config/config.go:29]: Why this high? Wouldn't `20` be enough?

File: finality-provider/config/config.go [hunk 3]
```diff
@@ -24,21 +24,21 @@ const (
...
+	defaultStatusUpdateInterval           = 20 * time.Second
+	defaultRandomInterval                 = 30 * time.Second
+	defautlUnbondingSigSubmissionInterval = 30 * time.Second
```
Comment (vitsalis) [inline finality-provider/config/config.go:32]: Does the finality provider need to submit unbonding sigs?

File: finality-provider/config/config.go [hunk 4]
```diff
@@ -24,21 +24,21 @@ const (
...
+	defaultFastSyncInterval               = 10 * time.Second
 	defaultFastSyncLimit                  = 10
-	defaultFastSyncGap                    = 6
+	defaultFastSyncGap                    = 3
 	defaultMaxSubmissionRetries           = 20
 	defaultBitcoinNetwork                 = "simnet"
 	defaultDataDirname                    = "data"
 	defaultDBPath                         = "bbolt-fpd.db"
-	defaultMaxNumFinalityProviders        = 3
+	defaultMaxNumFinalityProviders        = 10
```
Comment (vitsalis) [inline finality-provider/config/config.go:41]: `3` is good enough no?

File: finality-provider/config/config.go [hunk 5]
```diff
@@ -47,9 +47,9 @@ var (
 	//   ~/Users/<username>/Library/Application Support/Fpd on MacOS
 	DefaultFpdDir = btcutil.AppDataDir("fpd", false)
 
-	defaultBTCNetParams       = chaincfg.SimNetParams
+	defaultBTCNetParams       = chaincfg.RegressionNetParams
 	defaultEOTSManagerAddress = "127.0.0.1:" + strconv.Itoa(eotscfg.DefaultRPCPort)
-	DefaultRpcListener        = "127.0.0.1:" + strconv.Itoa(DefaultRPCPort)
+	DefaultRpcListener        = "0.0.0.0:" + strconv.Itoa(DefaultRPCPort)
```
Comment (vitsalis) [inline finality-provider/config/config.go:52]: Why this change?

File: finality-provider/config/config.go (hunk 2 above)
Comment (gitferry) [reply finality-provider/config/config.go:29]: Yeah I think so. Haven't tested it though. Previously when we have max gas limit of 50M and 100 validator submitting randomness at the same time, a block can only process 4 randomness. So we need 25 blocks to finish all the randomness submissions. Maybe we use 30?

File: finality-provider/config/config.go (hunk 3 above)
Comment (gitferry) [reply finality-provider/config/config.go:32]: Legacy code. Needs to be cut.

**Review by vitsalis — APPROVED** [review]

File: itest/test_manager.go [hunk 6]
```diff
@@ -39,14 +39,14 @@ import (
...
-	btcNetworkParams      = &chaincfg.SimNetParams
+	btcNetworkParams      = &chaincfg.SigNetParams
 
 	fpNamePrefix  = "test-fp-"
 	monikerPrefix = "moniker-"
 	chainID       = "chain-test"
 	passphrase    = "testpass"
 	hdPath        = ""
-	simnetParams  = &chaincfg.SimNetParams
+	simnetParams  = &chaincfg.SigNetParams
```
Comment (vitsalis) [inline itest/test_manager.go:49]:
```suggestion
	signetParams  = &chaincfg.SigNetParams
```
